### PR TITLE
Remove unnecessary and unused imports (use statements)

### PR DESCRIPTION
**Pull request by izharaazmi**
### Summary of Changes
Removed all occurrences of `use` statements that were either not used or not necessary being in the same namespace as the file.

### Testing Instructions
Code review should be good enough. Full file review would be needed though and not just the changed lines so that we can find if the removed imports are indeed unnecessary.

### Documentation Changes Required
None

### CODE REVIEW

File: build/helpTOC.php [hunk 1]
```diff
@@ -40,10 +40,6 @@
 // Load the admin en-GB.ini language file to get the JHELP language keys
 Factory::getLanguage()->load('joomla', JPATH_ADMINISTRATOR, null, false, false);
 
-// Import namespaced classes
-use Joomla\CMS\Version;
```
Comment (mbabker) [inline build/helpTOC.php:44]: These are used...

File: libraries/vendor/joomla/di/src/ContainerAwareTrait.php [hunk 2]
```diff
@@ -8,8 +8,6 @@
 
 namespace Joomla\DI;
 
-use Joomla\DI\Container;
```
Comment (mbabker) [inline libraries/vendor/joomla/di/src/ContainerAwareTrait.php]: This should be fixed in the Framework repo, not here.

File: build/helpTOC.php (hunk 1 above)
Comment (izharaazmi) [reply build/helpTOC.php:44]: Yes, imported in line 15 already.

Comment (mbabker) [reply build/helpTOC.php:44]: Oh.

File: libraries/vendor/joomla/di/src/ContainerAwareTrait.php (hunk 2 above)
Comment (izharaazmi) [reply libraries/vendor/joomla/di/src/ContainerAwareTrait.php]: Well right. Went in the flow 😄

File: administrator/components/com_languages/helpers/multilangstatus.php [hunk 3]
```diff
@@ -8,7 +8,6 @@
  */
 
 defined('_JEXEC') or die;
-use Joomla\Registry\Registry;
```
Comment (csthomas) [inline administrator/components/com_languages/helpers/multilangstatus.php:11]: It should be used at:

https://github.com/izharaazmi/joomla-cms/blob/532cd4f4be5eb4cb56f7e22de5cb1d69bd1473af/administrator/components/com_languages/helpers/multilangstatus.php#L269

**Comment by ghost** [comment]
> Code review should be good enough. 

@Quy something for you?

**Comment by brianteeman** [comment]
It has been a very long time since this PR was submitted and as there has been no response to the comments and no tests I am closing this as an abandoned PR. It can always be reopened if updated

**Pull request closed without merging**